pkg/kotsadm/objects/images.go

```diff
@@ -18,22 +18,29 @@ func GetAdminConsoleImages(deployOptions types.DeployOptions) map[string]string
 	postgresTag := getPostgresTag(deployOptions)
 	dexTag, _ := image.GetTag(image.Dex)
 
-	if deployOptions.KotsadmOptions.OverrideVersion != "" {
-		minioTag = deployOptions.KotsadmOptions.OverrideVersion
-		postgresTag = deployOptions.KotsadmOptions.OverrideVersion
-		dexTag = deployOptions.KotsadmOptions.OverrideVersion
-	}
-
 	minioImage := fmt.Sprintf("minio/minio:%s", minioTag)
 	postgresImage := fmt.Sprintf("postgres:%s", postgresTag)
 	dexImage := fmt.Sprintf("kotsadm/dex:%s", dexTag)
 
 	if s := kotsadmversion.KotsadmPullSecret(deployOptions.Namespace, deployOptions.KotsadmOptions); s != nil {
+		if deployOptions.KotsadmOptions.OverrideVersion != "" {
```

Comment:
Were there cases where deployOptions.KotsadmOptions was nil and that prompted the move into these two new spots?

Reply:
i was trying to point the tag to the nightly but that overrode the minio, postgres and dex tag